yaptide/routes/batch_routes.py

```diff
@@ -164,17 +165,22 @@ def delete(user: KeycloakUserModel):
                                     })
 
         cluster = fetch_cluster_by_id(cluster_id=simulation.cluster_id)
-
-        result, status_code = delete_job(simulation=simulation, user=user, cluster=cluster)
+        if to_fetch:
+            result, status_code = fetch_results_without_stopping(simulation=simulation,
```

Comment:
why don't you move that functionality to a different endpoint than delete ?
if you fetch results and stop simulation thats OK, but if you fetch results and continue it doesn't make sense to keep that in `delete` enpoint

Reply:
If you plan only to stop simulation and fetch the results then you should name the function not  `fetch_results_without_stopping`, but something like `stop_and_fetch_results`

Reply:
So you want to make 2 new endpoints for cancel simulation with data fetching, one for celery and one for batch? I'm asking because making new endpoint only for a batch simulation in my opinion could provide more dificulties than positives.

Reply:
I am just worried about consistency of naming of all the items we have: endpoints, functions etc...
If the only aim of this PR is to introduce the feature of "get the result" on cancel, then why do you name the function `fetch_results_without_stopping` ? 
Don't follow blindly any of my advice, just ensure that the names of variables, functions etc are meaningful and consistent.

Reply:
If you consider only "cancel and get the results" then one endpoint (delete) should be OK